src/lib.rs

```diff
@@ -337,7 +337,7 @@ pub struct Type {
 }
 
 /// Enum with additional information, depending on the kind of type.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
We don't want this struct to be clone-able. Let's find another way of getting the logic you are implementing here

Reply:
Hmm, okay.

Reply:
Fixed.